Games/types/Mafia/const/MeetingPriority.js

```diff
@@ -0,0 +1,9 @@
+const HOT_SPRINGS_MEETING_PRIORITY = 1
+const BANQUET_MEETING_PRIORITY = 2
+const JAIL_MEETING_PRIORITY = 3
+
```

Comment:
to avoid confusing with the night priority, how about adding the word "exclusive" somewhere so that it's clear that these are for meeting exclusivity?